packages/tools/devtools/devtools/package.json

```diff
@@ -69,7 +69,7 @@
 		"@fluidframework/fluid-static": "workspace:~"
 	},
 	"devDependencies": {
-		"@fluid-experimental/devtools-previous": "npm:@fluid-experimental/devtools@2.0.0-internal.7.2.0",
+		"@fluid-experimental/devtools-previous": "npm:@fluid-experimental/devtools@2.0.0-internal.8.0.0",
```

Comment:
You'll want to merge main and resolve the conflicts from https://github.com/microsoft/FluidFramework/pull/18820/files#diff-9ab0f7238002297f9e6660bb0180a0134ad9bfa3755d444dae28ed96dd12b702